docs-client/src/lib/specification.tsx

```diff
@@ -240,25 +240,8 @@ export class Specification {
         }
       }
     }
-
-    // TODO(trustin): Handle the docstrings of enum values.
-    for (const enm of this.data.enums) {
-      enm.docString = this.removeParamDocStrings(enm.docString as string);
-    }
-
     this.updateStructDocStrings(this.data.structs);
     this.updateStructDocStrings(this.data.exceptions);
-
-    this.data.enums.forEach(this.renderDocString);
-    this.data.exceptions.forEach(this.renderDocString);
-    for (const service of this.data.services) {
-      this.renderDocString(service);
-      for (const method of service.methods) {
-        this.renderDocString(method as HasDocString);
-        method.parameters.forEach(this.renderDocString);
-      }
-    }
-    this.data.structs.forEach(this.renderDocString);
   }
 
   private updateStructDocStrings(structs: Struct[]) {
```

Comment:
@minwoox I think `parseParamDocStrings`,`updateStructDocStrings` and `updateDocStrings` also can be removed (Because of the `Description` component). What do you think of that?

Reply:
I think we need that. 😅 
@jrhee17 will leave a comment about this, so please hold a minute. 😉

Reply:
Thanks @minwoox  😅  Sorry about the late comment.

I think the previous behavior was as follows:
> If a javadoc exists, then extract the `@param` values and inject these values as the parameters.

I think this feature is useful, especially for use-cases where teams want to document both code-level and `DocService` based on javadocs only. Also, I believe `@Description` is more explicit, and should have a higher priority than the `javadoc @param` based method.

What do you think of modifying the priority such that..
1. If a `@Description.docString` for a parameter/field doesn't exist, check the method/struct for `javadoc @param` and fill it if possible.
2. Otherwise, just use the original `docString`.

Additional notes:
@minwoox also suggested this logic can be moved to the java side in the long term (and I agree with this idea).
I think a separate PR can be uploaded as a follow-up, since I think the above logic change isn't too complex.

Reply:
Thank you for a detailed explanation.
I didn't expected javaDoc specification.
I also agreed `this logic can be moved to the java side in the long term`.

First, I will rollback removed code `parseParamDocStrings`,`updateStructDocStrings` and `updateDocStrings`.
And then, I will send separate PR 👍